content/pages/homepage.html

```diff
@@ -36,9 +36,10 @@ <h1 class="ui huge header">
                         data-analytics="signup-modal"
                          onclick="jQuery('#homepage-modal').modal('show');">
                         <i class="fad fa-play icon"></i>
-                        Get started
+                        Sign up now
                       </a>
-                      <a href="/about/" class="ui large stackable button">Learn more</a>
+                      <!-- Don't have multiple CTAs -->
+                      <!-- <a href="/features/" class="ui large stackable button">Learn more</a> -->
```

Comment:
Or just remove this button too, that's also fine

```suggestion
                      {# Don't have multiple CTAs #}
                      {# <a href="/features/" class="ui large stackable button">Learn more</a> #}
```